addons/website/views/snippets/snippets.xml

```diff
@@ -1149,14 +1150,17 @@
                     <t t-set="data_fields">[&quot;code&quot;]</t>
                 </t>
             </t>
-            <t t-call="website.snippet_options_conditional_visibility">
-                <t t-set="option_name">⌙ Languages</t>
-                <t t-set="attribute_rule">visibilityValueLangRule</t>
-                <t t-set="save_attribute">visibilityValueLang</t>
-                <t t-set="attribute_name">lang</t>
-                <t t-set="model">res.lang</t>
-                <t t-set="call_with">code</t>
-                <t t-set="data_fields">[&quot;code&quot;]</t>
+            <t t-if="len(current_website.language_ids) > 1">
+                <t t-call="website.snippet_options_conditional_visibility">
+                    <t t-set="option_name">⌙ Languages</t>
+                    <t t-set="attribute_rule">visibilityValueLangRule</t>
+                    <t t-set="save_attribute">visibilityValueLang</t>
+                    <t t-set="attribute_name">lang</t>
+                    <t t-set="model">res.lang</t>
+                    <t t-set="call_with">code</t>
+                    <t t-set="domain">[["website_ids", "in", [<t t-out="current_website.id"/>]]]</t>
```

Comment:
id in current_website.language_ids and no need of the new field?

Reply:
Way better !